### PR TITLE
chart: update image version and bump chart version

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-event-sources
-version: 0.1.0
-appVersion: 0.2.0
+version: 0.2.0
```
Comment (antoineco) [inline chart/Chart.yaml:3]: Considering the only change in the  chart is a fixed typo, maybe 0.1.1 is more aligned with the definition of semver?

File: chart/questions.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ questions:
     type: string
     label: Image Repository
   - variable: image.tag
-    default: "v0.2.0"
+    default: "v0.3.0"
```
Comment (antoineco) [inline chart/questions.yml:28]: Quotes are not needed.

File: chart/values.yaml [hunk 3]
```diff
@@ -15,31 +15,31 @@ imagePullSecrets: []
 image:
   registry: gcr.io
   repository: triggermesh/aws-event-sources-controller
-  tag: "v0.2.0"
+  tag: "v0.3.0"
```
Comment (antoineco) [inline chart/values.yaml:18]: Quotes are not needed.

File: chart/Chart.yaml (hunk 1 above)
Comment (sameersbn) [reply chart/Chart.yaml:3]: The controller and adapter image versions is updated in this chart version from `v0.2.0` to `v0.3.0`  (it's not a typo)

Comment (antoineco) [reply chart/Chart.yaml:3]: The component are 0.3.0, but the chart is versioned individually from the appVersion.

Comment (sameersbn) [reply chart/Chart.yaml:3]: okay. that makes sense. i'll bump the patch version instead

File: chart/Chart.yaml [hunk 4]
```diff
@@ -13,7 +13,7 @@ keywords:
   - triggermesh
   - serverless
 maintainers:
-  - name: sameersbn
-  - name: antoineco
+  - name: TriggerMesh Developers
+    email: dev@triggermesh.com
```
Comment (sameersbn) [inline chart/Chart.yaml:17]: note: added this extra change

File: chart/Chart.yaml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-event-sources
-version: 0.1.0
-appVersion: 0.2.0
+version: 0.0.1
```
Comment (antoineco) [inline chart/Chart.yaml:3]: Well now that's a downgrade 😱🙂

Comment (sameersbn) [reply chart/Chart.yaml:3]: haha.. my bad!

Comment (sameersbn) [reply chart/Chart.yaml]: fixed

**Comment by sameersbn** [comment]
@antoineco added some more typo fixes

**Pull request merged**